packages/design-system-twrnc-preset/package.json

```diff
@@ -64,7 +67,6 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/design-tokens": "^4.1.0",
```

Comment:
"@metamask/design-tokens" was incorrectly listed as a peer dependency here. It is used inside of the `design-system-twrnc-preset` which makes it a dependency 
- https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-system-twrnc-preset/src/twrnc-settings/typography.ts#L2
- https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-system-twrnc-preset/src/twrnc-settings/colors.ts#L1